stack-lts-12.26.yaml

```diff
@@ -0,0 +1,20 @@
+resolver: lts-12.26
+extra-deps:
+  - bugsnag-hs-0.2.0.8
+  - ua-parser-0.7.7.0
+  - bugsnag-1.0.0.1
+  - bugsnag-wai-1.0.0.1
+  - hspec-2.8.1
+  - hspec-core-2.8.1
+  - hspec-discover-2.8.1
+  - hspec-junit-formatter-1.1.0.1
+  - memcache-0.2.0.1
+  - scientist-0.0.0.0
+
+  - http-conduit-2.3.5
+  - monad-logger-0.3.31
+  - retry-0.8.1.0
+
+  - QuickCheck-2.13.2
+  - random-1.1
+  - splitmix-0.0.5
```

Comment:
Probably don't want all this stuff in the template.

Reply:
Some of this is likely required by `freckle-app` which is in the `tests` dependencies.